lib/Common/Memory/Recycler.h

```diff
@@ -1619,8 +1654,10 @@ class Recycler
     size_t TryMarkArenaMemoryBlockList(ArenaMemoryBlock * memoryBlocks);
     size_t TryMarkBigBlockList(BigBlock * memoryBlocks);
 #if ENABLE_CONCURRENT_GC
+#if FALSE // REVIEW: remove this code since not using
     size_t TryMarkBigBlockListWithWriteWatch(BigBlock * memoryBlocks);
```

Comment:
the caller is comment out with #if FALSE, and saying 'Re-enable this in the future.' I'm just removing the definition. let me add a TODO on the caller and investigate in the future

---
In reply to: [88136880](https://github.com/Microsoft/ChakraCore/pull/1968#discussion_r88136880) [](ancestors = 88136880)